sites/mosul/configs/openmrs/initializer_config/ampathforms/F01-MHPSS_Baseline.json

```diff
@@ -1918,12 +1918,423 @@
       "label": "Scores",
       "sections": [
         {
-          "label": "MHOS (required) - In the past week…",
+          "label": "MHOS - Mental Health Outcome Score",
           "isExpanded": false,
           "questions": [
             {
-              "id": "problemsPerformingDailyActivities",
-              "label": "Did you have problems performing daily activities such as cooking bathing or chores outside the household?",
+              "id": "inThePastWeek",
+              "label": "In the past week",
+              "type": "obs",
+              "required": false,
+              "questionOptions": {
+                "rendering": "markdown",
+                "concept": "inThePastWeek"
+              },
+              "hide": {
+                "hideWhenExpression": "age < 15"
+              }
+            },
+            {
+              "id": "didYouHaveProblemsPerformingDailyActivitiesSuchAsCookingBathingOrChoresOutsideTheHousehold",
```

Comment:
You need to pay attention to changes in the `id` because it might have unintended side effects to other parts of the form